source/clear-linux/get-started/virtual-machine-install/virtualbox.rst

```diff
@@ -3,88 +3,79 @@
 Run pre-configured |CL-ATTR| as a VirtualBox\* Virtual Machine
 ##############################################################
 
-VirtualBox\* is a type 2 hypervisor from Oracle. This document explains how 
-to create a virtual machine on the `VirtualBox hypervisor`_  with |CL-ATTR| 
+VirtualBox\* is a type 2 hypervisor from Oracle. This document explains how
+to create a virtual machine on the `VirtualBox hypervisor`_  with |CL-ATTR|
 as the guest operating system.
 
-These instructions make use of a preinstalled |CL| disk image to setup a |CL| 
-virtual machine without manual installation. |CL| can also be installed from 
+These instructions make use of a preinstalled |CL| disk image to setup a |CL|virtual machine without manual installation. |CL| can also be installed from
 scratch on a |VB| using the |CL| installed.
 See: :ref:`virtualbox-cl-installer`
 
 .. contents:: :local:
     :depth: 2
 
-
-
 .. _vbox-prereqs-begin:
-
+gi
 Prerequisites
 *************
 
-Before continuing make sure that you have: 
+Before continuing make sure that you have:
 
-#. Enabled virtualization, such as Intel® 
-   `Virtualization Technology`_ (Intel® VT), on the host system from 
+#. Enabled virtualization, such as Intel®
+   `Virtualization Technology`_ (Intel® VT), on the host system from
    EFI/BIOS.
 
-#. Downloaded and installed |VB| **version 6.0 or greater** from 
-   the `official VirtualBox website`_ per the  `appropriate instructions`_ 
+#. Downloaded and installed |VB| **version 6.0 or greater** from
+   the `official VirtualBox website`_ per the  `appropriate instructions`_
    for your platform.
 
-.. _vbox-prereqs-end:
-
+.. todo: Reference to `appropriate instructions` contradicts intro using CL as host OS
 
+.. _vbox-prereqs-end:
 
 Download and extract |CL|
 *************************
 
-The |CL| live image needs to be downloaded and extracted. The live image will 
-be used to created a |VB| virtual disk image that can be used with a 
+The |CL| live iso needs to be downloaded and extracted. The live image will
+be used to created a |VB| virtual disk image that can be used with a
 :abbr:`VM (Virtual Machine)`.
 
-#. Download the **live image** (:file:`clear-<VERSION-live.img.xz`) of
-   |CL|. On the `downloads page`_, this is listed as 
-   **Clear Linux OS live boot image**.
-   
-   
-   You can also use this command to download from a terminal: 
-
-   .. code-block:: bash
+.. todo: restart edits here 05.17.19
 
-      curl -O https://cdn.download.clearlinux.org/image/$(curl https://cdn.download.clearlinux.org/image/latest-images | grep live.img)
+#. Download the **live image** (:file:`clear-<VERSION-live-server.iso.xz`) of
+   |CL|. On the `downloads page`_, this is listed as
+   **Clear Linux OS live boot image**.
```

Comment:
This takes me to the downloads page and there is no entry for **Clear Linux OS live boot image**